integration/client/client.go

```diff
@@ -434,7 +434,7 @@ func (c *Client) GetRules(ctx context.Context) (*RulesResponse, error) {
 	resp := RulesResponse{}
 	err = json.Unmarshal(buf, &resp)
 	if err != nil {
-		return nil, fmt.Errorf("error parsing response data: %w", err)
+		return nil, fmt.Errorf("error parsing response data %q: %w", buf, err)
```

Comment:
Q: given this buffer contains results of `GetRules`. can this be huge enough to cause trouble? (also can it make too verbose error message?)